topics/admin/tutorials/ansible/tutorial.md

```diff
@@ -69,14 +69,28 @@ Looking at each of these briefly:
 
 ```ini
 [webservers]
-web_a
-web_b
+web_1.host.fqdn.or.ip
+web_2.host.fqdn.or.ip
```

Comment:
maybe we show a hostname example below, maybe just use an IP here. [ip ranges for docs](https://tools.ietf.org/html/rfc5737)

Reply:
Done. And yes, I think it's better this way.